WarehouseManagement.Api/Program.cs

```diff
@@ -22,6 +23,24 @@
 builder.Services.AddApplicationIdentity(builder.Configuration);
 builder.Services.AddApplicationService();
 
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+ .AddJwtBearer(options =>
+ {
+     options.TokenValidationParameters = new TokenValidationParameters
+     {
+         ValidIssuer = builder.Configuration["Jwt:Issuer"],
+         ValidAudience = builder.Configuration["Jwt:Audience"],
+         IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(builder.Configuration["Jwt:Key"]!)),
```

Comment:
i guess we need values for these inside the appsettings?

Reply:
We currently have:
![image](https://github.com/user-attachments/assets/96edf815-78d0-43ae-9067-629f7e23ddd2)